src/main/java/gndata/lib/util/change/Merge.java

```diff
@@ -0,0 +1,65 @@
+package gndata.lib.util.change;
+
+import java.util.*;
+
+import com.hp.hpl.jena.ontology.OntModel;
+import com.hp.hpl.jena.rdf.model.*;
+import com.hp.hpl.jena.vocabulary.*;
+
+/**
+ * Class that knows how to update Metadata entities.
+ */
+public class Merge implements Change {
+
+    private Model new_object;
+    private OntModel ontology;
+    private boolean applied = false;
+
+    private Model changes = ModelFactory.createDefaultModel();
```

Comment:
An assumption is that changes can be accessed later, maybe even publicly. This is just implementation step 1)